.github/workflows/ci.yml

```diff
@@ -607,37 +657,25 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4
-      - name: Check which files were touched
-        id: changed-files
-        uses: step-security/changed-files@95b56dadb92a30ca9036f16423fd3c088a71ee94 # v46
```

Comment:
I realized changes in `@scalar/themes` and other dependencies can affect components too so we probably just want to run this all the time?